### PR TITLE
NAS-107513 / 12.0 / Clarify overprovision size field name

### PR DESCRIPTION
**Pull request by erinix**
NAS-107513

### CODE REVIEW

File: src/app/helptext/system/advanced.ts [hunk 1]
```diff
@@ -47,7 +47,7 @@ export const helptext_system_advanced = {
   swapondrive_validation: [ Validators.required, Validators.min(0), Validators.max(99) ],
 
   overprovision: {
-    placeholder: T('Overprovision Size in GiB'),
+    placeholder: T('SLOG Overprovision Size in GiB'),
     tooltip: T('Overprovisioning can increase the performance and lifespan of SSDs. \
  Defining a number of GiB here overprovisions disks during pool creation. \
  Examples: 50 GiB, 10g, 5GB'),
```
Comment (Mrt134) [inline src/app/helptext/system/advanced.ts:53]:
```suggestion
    tooltip: T('Overprovisioning a Separate Log (SLOG) SSD can increase its performance and lifespan by \
 distributing writes and erases across more drive flash blocks. \
 Defining a number of GiB here overprovisions SLOG disks during pool creation. \
 Examples: 50 GiB, 10g, 5GB'),
```

**Review by Mrt134 — COMMENTED** [review]
Added a quick rework of the related help text

File: src/app/helptext/system/advanced.ts [hunk 2]
```diff
@@ -47,9 +47,10 @@ export const helptext_system_advanced = {
   swapondrive_validation: [ Validators.required, Validators.min(0), Validators.max(99) ],
 
   overprovision: {
-    placeholder: T('Overprovision Size in GiB'),
-    tooltip: T('Overprovisioning can increase the performance and lifespan of SSDs. \
- Defining a number of GiB here overprovisions disks during pool creation. \
+    placeholder: T('SLOG Overprovision Size in GiB'),
+    tooltip: T('Overprovisioning a Separate Log (SLOG) SSD can increase its performance and lifespan by \
+ distributing writes and erases across more drive flash blocks. \
+ Defining a number of GiB here overprovisions SLOG disks during pool creation. \
```
Comment (Mrt134) [inline src/app/helptext/system/advanced.ts:53]:
```suggestion
    placeholder: T('LOG (Write Cache) Overprovision Size in GiB'),
    tooltip: T('Overprovisioning a ZFS Log SSD can increase its performance and lifespan by \
 distributing writes and erases across more drive flash blocks. \
 Defining a number of GiB here overprovisions ZFS Log disks during pool creation or extension. \
```

**Review by Mrt134 — APPROVED** [review]
Rework a little from William feedback.

**Review by dkmullen — APPROVED** [review]

**Pull request merged**